pkg/blackfriday/block_test.go

```diff
@@ -733,8 +733,8 @@ func TestOrderedList(t *testing.T) {
 		"1. List\n\n          code block with spaces\n",
 		"<ol>\n<li><p>List</p>\n\n<pre><code>  code block with spaces\n</code></pre></li>\n</ol>\n",
 
-		"1. List\n    * Mixted list\n",
-		"<ol>\n<li>List\n\n<ul>\n<li>Mixted list</li>\n</ul></li>\n</ol>\n",
+		"1. List\n    * Mixed list\n",
+		"<ol>\n<li>List\n\n<ul>\n<li>Mixed list</li>\n</ul></li>\n</ol>\n",
```

Comment:
This file is strange. I couldn't quite figure out what it was trying to do.

I left `<div>..</divv>` alone because it looked like it cared that the latter part misspelled `div`. -- Note that if you don't care about how you misspell a thing, then `<div>...</dive>` would work just as well and not upset spell checkers.

Here it doesn't appear to care about the spelling, so it didn't seem worth preserving.